sigma/backends/panther/helpers/python_helper.py

```diff
@@ -0,0 +1,101 @@
+from typing import Any, Union
+
+import yaml
+from sigma.conditions import ConditionAND, ConditionFieldEqualsValueExpression, ConditionOR
+from sigma.conversion.state import ConversionState
+
+from sigma.backends.panther.helpers.base import BasePantherBackendHelper
+
+
+class PythonHelper(BasePantherBackendHelper):
+    @staticmethod
+    def invert_if_needed(func):
+        def inner(helper, cond, state):
+            negated = getattr(cond, "negated", False)
+            if negated:
+                result = "not " + func(helper, cond, state)
+            else:
+                result = func(helper, cond, state)
+            return result
+
+        return inner
+
+    @staticmethod
+    def get_key_path_value(path: str):
+        key_path = '"' + '", "'.join(path.split(".")) + '"'
+        key_path_value = f"deep_get(event, {key_path})"
+        return key_path_value
+
+    @invert_if_needed
+    def convert_condition_as_in_expression(
+        self, cond: Union[ConditionOR, ConditionAND], state: ConversionState
+    ) -> Any:
+        keys = [x.field for x in cond.args]
+
+        assert len(keys) and len(set(keys)) == 1
+        return f"{self.get_key_path_value(keys[0])} in {[x.value.to_plain() for x in cond.args]}"
+
+    @invert_if_needed
+    def convert_condition_field_eq_val_str(
+        self, cond: ConditionFieldEqualsValueExpression, state: ConversionState
+    ) -> Any:
+        key_path = self.get_key_path_value(cond.field)
+        value = str(cond.value)
+        regexp_special_chars = ".+*?^$()[]{}|\\"
+        if "*" in value:
+            regex_value = ""
+            for i in value:
+                if i == "*":
+                    regex_value += ".*"
+                elif i in regexp_special_chars:
+                    regex_value += f"\\{i}"
+                else:
+                    regex_value += i
+            return f're.match("{regex_value}", {key_path})'
+
+        return f'{key_path} == "{value}"'
+
+    @invert_if_needed
+    def convert_condition_field_eq_val_num(
+        self, cond: ConditionFieldEqualsValueExpression, state: ConversionState
+    ) -> Any:
+        return f'{self.get_key_path_value(cond.field)} == "{cond.value.to_plain()}"'
+
+    @invert_if_needed
+    def convert_condition_field_eq_val_null(
+        self, cond: ConditionFieldEqualsValueExpression, state: ConversionState
+    ) -> Any:
+        return f"{self.get_key_path_value(cond.field)} is None"
+
+    @invert_if_needed
+    def convert_condition_field_eq_val_re(
+        self, cond: ConditionFieldEqualsValueExpression, state: ConversionState
+    ) -> Any:
+        key_path = self.get_key_path_value(cond.field)
+        value = str(cond.value.regexp)
```

Comment:
I don't think this is escaping special regex chars properly for python.  The escapes in line 51 look correct, with `\\` as the python regex escape character.  This line is generating regex that looks like this `re.match(".*\\AtBroker\.exe"` where it should be `re.match(".*\\\AtBroker\\.exe"`